quaiclient/quaiclient.go

```diff
@@ -258,3 +266,95 @@ func (ec *Client) SetSyncTarget(ctx context.Context, header *types.Header) {
 	fields := header.RPCMarshalHeader()
 	ec.c.CallContext(ctx, nil, "quai_setSyncTarget", fields)
 }
+
+//// Miner APIS
+
+// GetPendingHeader gets the latest pending header from the chain.
+func (ec *Client) GetPendingHeader(ctx context.Context) (*types.Header, error) {
+	var pendingHeader *types.Header
+	err := ec.c.CallContext(ctx, &pendingHeader, "quai_getPendingHeader")
+	if err != nil {
+		return nil, err
+	}
+	return pendingHeader, nil
+}
+
+// ReceiveMinedHeader sends a mined block back to the node
+func (ec *Client) ReceiveMinedHeader(ctx context.Context, header *types.Header) error {
+	data := header.RPCMarshalHeader()
+	return ec.c.CallContext(ctx, nil, "quai_receiveMinedHeader", data)
+}
+
+// Filters
+
+// SubscribeFilterLogs subscribes to the results of a streaming filter query.
+func (ec *Client) SubscribeFilterLogs(ctx context.Context, q quai.FilterQuery, ch chan<- types.Log) (quai.Subscription, error) {
+	arg, err := toFilterArg(q)
+	if err != nil {
+		return nil, err
+	}
+	return ec.c.EthSubscribe(ctx, ch, "logs", arg)
```

Comment:
Does this use quai namespace or eth namespace?

Reply:
it should be quai, good catch

Reply:
Updated it